src/SearchIndex.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * This file is part of AlgoliaSearch Client PHP Helper.
+ *
+ * (c) Algolia Team <contact@algolia.com>
+ *
+ *  For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ */
+
+namespace Algolia\AlgoliaSearch\Helper;
+
+use Algolia\AlgoliaSearch\Exceptions\NotFoundException;
+
+final class SearchIndex extends \Algolia\AlgoliaSearch\SearchIndex
+{
+    /**
+     * Check if the index exist.
+     *
+     * @return bool
+     */
+    public function exists()
```

Comment:
This feature is now ported to the php client. It can be removed from here. And also the tests associated.